cobalt/media/media_module.cc

```diff
@@ -219,6 +219,12 @@ bool MediaModule::SetConfiguration(const std::string& name, int32 value) {
                 << (value ? "true" : "false");
       return true;
     }
+  } else if (name == "PlayerConfiguration.SetMaximumNumberOfSamplesPerWrite" &&
```

Comment:
Would it be more straightforward to replace Media.EnableBatchedSampleWrite with Media.MaximumNumberOfSamplesPerWrite that controls the number from the Cobalt side, and let Android TV returns 256 by defaults?  The value should be default to 1.

The web app can experiment with any values on Android TV (or other platforms), and we don't have to plumb it through the Starboard interface.

Reply:
We will wait a PR using a general way to set/get value via H5VCC for media experiments, and then replace Media.EnableBatchedSampleWrite with Media.MaximumNumberOfSamplesPerWrite .

Reply:
Refactor "EnableBatchedSampleWrite" to "AudioBatchedSampleWrite".